Scripts/semantic-convention/templates/SemanticAttributes.swift.j2

```diff
@@ -76,14 +83,21 @@ public enum {{enum}}: String {
     /**
     {{attribute.brief | to_doc_brief}}.
     */
+    /// - Tag: otel{{class_name}}
```

Comment:
I have absolutely no idea why we must use triple slash syntax for tags, but everything else works with `/**`.

Tags do not work unless you use triple slash syntax.
I have verified this functionality, both work side by side.